docs/source/meta/planning.rst

```diff
@@ -12,7 +12,7 @@ The goal of the DeltaMetrics project is to provide convenient tools to analyze s
 Objectives
 ----------
 
-#. Users should be able to instantiate the Cube object with a numpy array where dimensions are expected to be ``x-y-v``, and a value specifying the type of data stored in ``v``. ``v`` could be, lidar scans, overhead photos, grain size maps (DeltaRCM), flow velocity (DeltaRCM), etc. An optional time array can be supplied, specifying time at each z; we can also accept a single value ``T``, and time is linearly spaced wrt ``T`` and ``0``.
+#. Users should be able to instantiate the Cube object with a numpy array where dimensions are expected to be ``t-x-y``, and a value specifying the type of data stored in the array. Data could be, lidar scans, overhead photos, grain size maps (DeltaRCM), flow velocity (DeltaRCM), etc. An optional time array can be supplied, specifying time at each z; we can also accept a single value ``T``, and time is linearly spaced wrt ``T`` and ``0``.
 #. Users should be able to extract ``Section`` s (e.g., :class:`~deltametrics.section.RadialSection`) from the ``Cube``. Several types of sectioning should be supported. By default, we should try to process to "stratigraphy" via an elevation timeseries attribute of the ``Cube``, but can have a flag to leave as raw section.
 #. Users should be able to do processed and compute statistics on the ``Section`` s. For example, idenfying channels (?) or computing compensation statistics (extract a ``Horizon`` first?). These routines can live in ``deltametrics.strat`` (:doc:`../reference/strat/index`) or in other modules.
```

Comment:
I like the framework of the `Cube` from which either Sections or Plans are extracted and metrics are computed. In this format, where would methods for 3-D or bulk stratigraphic metrics be computed (e.g. net-gross)? It makes me wonder if the `Section` should be named `Stratigraphy` or something else? Unless those 3-D stratigraphic metrics are just going to be housed in `Cube` (which is fine by me).

Reply:
I *think* what we should do, is have the modules `strat.py` and `channel_network.py` or `overland_flow.py` or whatever that have all the fancy algorithms, computations, statistics, etc. Leave `cube.py` `section.py` and `plan.py` to set up the objects that these algorithms take as arguments. We could "wrap" a few of the most commonly used ones with methods of the object classes too, if needed. 

I mentioned somewhere in a docstring about porting the strat calculations out to a module. This may be the right move, but we can always do it in the future.